include/oneapi/mkl/sparse_blas/matrix_view.hpp

```diff
@@ -38,10 +38,6 @@ struct matrix_view {
     matrix_descr type_view = matrix_descr::general;
     uplo uplo_view = uplo::lower;
     diag diag_view = diag::nonunit;
-
-    matrix_view() = default;
-
-    matrix_view(matrix_descr type_view) : type_view(type_view) {}
```

Comment:
@Rbiessy, @pgorlani, I somehow missed this PR back when it was created. I reviewed the PR, but it seems to be going in the opposite direction of https://github.com/uxlfoundation/oneAPI-spec/issues/578 by removing constructors rather than adding them. But at the same time, it also does seem to solve the issue https://github.com/uxlfoundation/oneAPI-spec/issues/578. I'm not sure I understand what you mean by "turn matrix_view into an aggregate", could you explain how this works, or point me to a link?

Reply:
Hey @gajanan-choudhary, if I remember correctly, removing explicit constructors allows for this type to be initialized in more ways, see https://en.cppreference.com/w/cpp/language/aggregate_initialization the section "Implicitly initialized elements".
I didn't push for this PR as we don't have the bandwidth to add new features on oneMath anymore. This would also need a spec update so I'd rather close this PR. Anyone picking up this task in the future can find the reference to this PR and take inspiration.